README.md

```diff
@@ -48,7 +63,7 @@ You'll probably want to include this file once in your app, ideally at the top-m
 **JS + webpack or similar**
 
 ```js
-import '@apollo/space-kit/reset.css';
+import "@apollo/space-kit/reset.css";
 ```
 
 **LESS**
```

Comment:
i know this isnt part of this pr, but the less import string has no closing quote right now